wiki/Modding/Normal_Mod/en.md

```diff
@@ -0,0 +1,14 @@
+---
+tags:
+  - NM
+---
+
+# Normal mod
+
+**Normal mods** (*NMs*) are [mods](/wiki/Modding/) that will be provided without requiring any work from the requesting beatmapper. Normal mods are usually done to get second opinions on a [beatmap](/wiki/Beatmaps/) and help improve it. The clarification word for normal mods might seem unnecessary, but modders use it to distinguish it from mod for mod exchanges.
```

Comment:
```suggestion
A **normal mod** (*NM*) is a [mod](/wiki/Modding/) that will be provided without requiring any work from the requesting beatmapper. Normal mods are usually done to get second opinions on a [beatmap](/wiki/Beatmaps) and help improve it. Many modders use the term "NM" to distinguish it from mod for mod exchanges.
```
> Abbreviations, acronyms, and initialisms should not be pluralised.

\- [ASC](https://osu.ppy.sh/help/wiki/Article_Styling_Criteria#abbreviations,-acronyms,-and-initialisms)